src/udev/udev-builtin-keyboard.c

```diff
@@ -172,16 +172,14 @@ static int builtin_keyboard(sd_device *dev, int argc, char *argv[], bool test) {
                 return log_device_error_errno(dev, r, "Failed to get device name: %m");
 
         FOREACH_DEVICE_PROPERTY(dev, key, value) {
```

Comment:
Here, `{}` is not necessary anymore.

Reply:
Removed. Although I think the {} improve readability for larger blocks.